.github/workflows/link_and_spell_check.yml

```diff
@@ -0,0 +1,48 @@
+on:
```

Comment:
In general, it is difficult to see how this flow is configured and it is not clear why certain requirements were chosen. For instance, what is the reason for choosing 80 characters instead of 100 or 130? Without a config checked in, it feels a bit like this limit was just chosen because it is the standard of some tool with no further thought to it.

Reply:
I think it is all in all helpful to stick with some rules across the board. The configuration hence lives on the org repo. Having a fixed limit on the line length makes it easier to see diffs when something is changed.